matlab/gtsam_tests/testUtilities.m

```diff
@@ -52,5 +52,5 @@
 values.insert(2, (13:18)');
 values.insert(7, Pose3());
 actual = utilities.extractVector(values);
-expected = reshape(1:18, 3, 6);
+expected = reshape(1:18, 6, 3)';
```

Comment:
Hmmm. It should be 3*6, right?

Reply:
matlab reshapes columns-first so we have to do 6, 3 then transpose

Just to be clear, `expected` will be:
```
1 2 3 4 5 6
7 8 9 10 11 12
13 14 15 16 17 18
```
This is what you're expecting, right?

Reply:
Yep, did not see the transpose... Sorry!